front/src/NotificationWindow.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "styled-components";
+import faTimes from "@fortawesome/fontawesome-free-solid/faTimes";
+import FontAwesomeIcon from "@fortawesome/react-fontawesome";
+
+const Window = styled.div`
+  background-color: ${props => props.backgroundColor};
```

Comment:
isnt it better to let the component render the color on its own?

Reply:
Background color now depends on `notificationType`